srcpkgs/hlint/files/stack.yaml

```diff
@@ -0,0 +1,8 @@
+resolver: lts-15.4
+packages:
+  - .
+extra-deps:
+  - ghc-lib-parser-8.10.1.20200412
```

Comment:
Nope, this should wait for ghc 8.10, this is also reason why lts 15.12 has still major version 2 of hlint.